pyriemann/utils/tangentspace.py

```diff
@@ -50,3 +57,14 @@ def untangent_space(T, Cref):
         covmats[i] = numpy.dot(numpy.dot(C12, covmats[i]), C12)
 
     return covmats
+
+
+def transport(Covs, Cref, metric='riemann'):
+    """Parallel transport of two set of covariance matrix.
```

Comment:
Parallel transport of two sets of covariance matrices.

also you have no docstring :)